yard-cucumber.gemspec

```diff
@@ -57,7 +57,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rake', '~> 10'
 
   s.add_dependency 'gherkin', '>= 4.0', '< 6.0'
-  s.add_dependency 'cucumber', '>= 2.0', '< 4.0'
+  s.add_dependency 'cucumber', '>= 2.0', '<= 5.0'
```

Comment:
This will allow 5.0.0, but not 5.0.1. I think it's better to use `< 6.0` in this case.